src/Notification/Type/DgtNotification.php

```diff
@@ -5,47 +5,47 @@
 class DgtNotification
 {
     /**
-     * @var string
+     * @var null | 'RequestStatusChange' | 'ProductStatusChange' | 'ProductDelivery' | 'CorrectionStatusChange'
      */
-    private $notificationType;
+    private $notificationType = null;
```

Comment:
even without assigning null, the default value is null

Reply:
This code is generated by the library, check the readme [here](https://github.com/openeuropa/epoetry-client/blob/2.x/README.md#generate-soap-related-classes).